gwt-regexp/pom.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.gwtproject.regexp</groupId>
     <artifactId>gwt-regexp</artifactId>
     <version>1.0-SNAPSHOT</version>
-    <packaging>jar</packaging>
+    <packaging>gwt-lib</packaging>
 
     <name>GWT Regular Expression</name>
     <description>TBD</description>
```

Comment:
populate this

Reply:
The previous configuration was copy the sources and the *.gwt.xml file manually, while making this gwt-lib and configure the plugin will copy the sources and the *.gwt.xml by default without the need to that manually. At the end, gwt-lib is just a Jar with sources inside so this doesn't change the usage yet it simplify the copy of the resources

Reply:
I think switching to gwt-lib means we need to also use the gwt-m-p. (correct me if I am wrong) This would something we wanted to avoid, cause it adds a dependency to GWT. I would like to stay with jar and copy manually the resources.

Reply:
I see your point here, we can remove it as it just copies the sources. Okay let me change this to manual copy.

Reply:
As long as we are producing jars with sources in them, we are not forcing users to switch to the new plugin, nor will this affect j2cl usage - and it will make it easier to use this dependency if someone _is_ using the new plugin.

Reply:
When I said "this", I meant the <description> - it should not read "TBD".

Reply:
The description should be changed.